src/constants.py

```diff
@@ -1,4 +1,44 @@
+ADD_LABELS = "add_labels"
+REMOVE_LABELS = "remove_labels"
+
 BLOCK_MERGE_VERIFY_CONTEXT = "Verified label"
+
 LABEL_VERIFIED = "Verified"
 LABEL_APPROVE = "Approve"
 NEEDS_MAINTAINERS_APPROVE = "Needs Maintainers Approve"
+
+LABEL_MAJOR = "major"
+LABEL_MINOR = "minor"
+LABEL_PATCH = "patch"
+
+LABEL_SEMVER_PREFIX = "Semver"
```

Comment:
Do we need `LABEL_SEMVER_PREFIX` in the labels? or can we leave it as `MAJOR`?

or, what about or major for example the label will be:
MAJOR/Breaking-Change

WDYT?

Reply:
I agree; `MAJOR/Breaking-Change` is clearer